src/main/java/de/bonndan/nivio/output/icons/IconService.java

```diff
@@ -30,7 +28,7 @@ public IconService(LocalIcons localIcons, ExternalIcons externalIcons) {
 
     /**
      * Returns the proper icon url for an item.
-     *
+     * <p>
```

Comment:
Please ask @basseur how to remove this by configuring your IDE.

Reply:
I figured it out myself. Credits go to Google... ;)